### PR TITLE
Correct Support Plate BC Constraining Rigid Motion

### LINKED ISSUE #413: Remove Rigid Body Motion from Support Plates
## Reason

Test failure for `sfr` regression tests prompted a look at support plates tests. These support plate tests allow for rigid body motion. As the max displacements are failing while the average strain is not; this rotational motion may explain the failures.

## Design

Add another node off set from center to constrain rotation.

## Impact

The regression test inputs will now correctly illustrate how to remove rigid body motion for more robust modeling.

### PR DESCRIPTION
**Pull request by elementx54**
Removed rigid body motion by adding another nodeset to apply a zero BC. This prevents the plate from spinning around the center node.

Closes #413

### CODE REVIEW

**Comment by elementx54** [comment]
I'd like to point out that the average strains in the gold files did not change even though the max displacements did change.

File: sfr/single_assembly/core_support_plate_2d.i [hunk 1]
```diff
@@ -28,6 +28,13 @@ Tsf    = 623.15 # stress-free temperature for the ComputeMeanThermalExpansionFun
     top_right = '1e-3 1e-3 0'
     bottom_left = '-1e-3 -1e-3 0'
   []
+  [add_outer_nodeset]
+    type = BoundingBoxNodeSetGenerator
+    input = add_nodeset
+    new_boundary = outer_node
+    top_right = '0.064 1e-3 0'
+    bottom_left = '0.058 -1e-3 0'
```
Comment (GiudGiud) [inline sfr/single_assembly/core_support_plate_2d.i:36]: so are we not constraining radial expansion by doing that though? and creating asymmetty?

Comment (gambka) [reply sfr/single_assembly/core_support_plate_2d.i:36]: If you pick the node in the same plane and along one of the coordinate axes as 0,0,0 and you fix it in the appropriate direction it should not create asymmetry. If you have perfect symmetry picking a node on the x-axis and fixing it against movement in the y direction doesn't constrain radial expansion. The node is free to move in x, which on the x-axis is the radial direction. For a circle, with symmetric conditions all the nodes on the x and y axis could be fixed (x-axis against moving in y, y-axis against moving in x) as they represent the symmetry plane. You'd get the same results if you modeled a quarter of the circle.  The problem as it stands now is free to rotate unconstrained about the origin.

**Comment by elementx54** [comment]
Did the tests boxes fail? Can someone restart these tests?

File: sfr/single_assembly/core_support_plate_2d.i (hunk 1 above)
Comment (elementx54) [reply sfr/single_assembly/core_support_plate_2d.i:36]: If you are in today, I could give you a quick explanation on a white board about this.

Comment (GiudGiud) [reply sfr/single_assembly/core_support_plate_2d.i:36]: I ll be there tomorrow

Comment (GiudGiud) [reply sfr/single_assembly/core_support_plate_2d.i:36]: but yes I understand thanks for the explanation

**Comment by GiudGiud** [comment]
we should be good here soon. Just needed a small regold

**Review by GiudGiud — APPROVED** [review]
BISON failures fixed. Thanks for the patch @elementx54 !

**Pull request merged**